### DISCUSSION TITLE
圖標點的LABEL第一個不顯示

### DISCUSSION THREAD

**Discussion by stan88** [zouyaoji/vue-cesium#362] [Q&A]
您好,
圖標點產生第一個時,
LABEL不出現
當按下第二個圖標點後就正常
請參考操作影片
謝謝

[Video_2023-02-22_161907.zip](https://github.com/zouyaoji/vue-cesium/files/10801497/Video_2023-02-22_161907.zip)

**Comment by zouyaoji** [comment]
https://zouyaoji.top/vue-cesium/#/zh-CN/component/analyses/vc-drawings

在线文档表现正常，未能重现。

**Comment by stan88** [comment]
您好,
目前用的是3.1.4版本
只有圖標點導致全部的LABEL全部消失
其他的實例不會
有個地方是第一次按下圖標點按鈕時,
VIEWER的CONTAINER會整個閃一下,
其餘繪制按鈕不會
清空全部繪制實例後,再按下繪制第一個圖標點時,
也會再出現全部的LABEL全部消失
請看影片

[Video_2023-02-24_110956.zip](https://github.com/zouyaoji/vue-cesium/files/10820314/Video_2023-02-24_110956.zip)

**Comment by stan88** [comment]
您好,昨天加今天再測試後,這問題己不見,謝謝。
